packages/vue/src/examples/pages/thank-you/ThankYou.vue

```diff
@@ -0,0 +1,164 @@
+      <div class="thank-you__banner-info">
+        <SfHeading title="It's ordered!" class="sf-heading--no-underline" />
+        <p class="thank-you__order-number">
+          Order No. <strong>#80932031-030-00</strong>
+        </p>
+      </div>
+    </div>
+    <div class="thank-you__wrapper">
+      <SfHeading
+        title="You've successfully placed the order"
+        class="thank-you__heading sf-heading--left"
+      />
+      <p class="thank-you__text">
+        You can check status of your order by using our delivery status feature.
+        You will receive an order confirmation e-mail with details of your order
+        and a link to track its progress.
+      </p>
+      <SfButton class="thank-you__notifications-button"
+        >ALLOW ORDER NOTIFICATIONS</SfButton
+      >
+      <div class="thank-you__contact">
+        <SfHeading
+          :level="3"
+          class="sf-heading--left sf-heading--no-underline"
+          title="Primary contacts for any questions"
+        ></SfHeading>
+        <p class="thank-you__text">
+          <span>{{ address.name }}</span
+          ><br />
+          <span>{{ address.street }}</span
+          ><br />
+          <span>{{ address.city }}</span
+          ><br /><br />
+          <span>{{ address.email }}</span>
+        </p>
+      </div>
+      <SfHeading
+        title="Your Account"
+        class="thank-you__heading sf-heading--left"
+      />
+      <p class="thank-you__text">
+        You can log to your account using e-mail and password defined earlier.
+        On your account you can edit your profile data, check history of
+        transactions, edit subscription to newsletter.
+      </p>
+      <SfHeading
+        title="What can we improve"
+        class="thank-you__heading sf-heading--left"
+      />
+      <p class="thank-you__text">
+        Your feedback is important to us. Let us know what we could improve.
+      </p>
+      <div class="thank-you__buttons">
+        <SfButton class="color-secondary sf-button--full-width"
+          >SEND MY FEEDBACK</SfButton
+        >
+        <SfButton
+          class="sf-button--outline color-secondary sf-button--full-width"
+          >BACK TO SHOP</SfButton
+        >
+      </div>
+    </div>
+  </div>
+</template>
+<script>
+import { SfHeading, SfButton } from "@storefront-ui/vue";
+export default {
+  name: "Product",
+  components: {
+    SfHeading,
+    SfButton
+  },
+  data() {
+    return {
+      address: {
+        name: "Divante Headquarter",
+        street: "St. Dmowskiego 17, 53-534",
+        city: "Wroclaw, Poland",
+        email: "demo@vuestorefront.io"
+      }
+    };
+  }
+};
+</script>
+<style lang="scss" scoped>
+@import "~@storefront-ui/vue/styles";
+@mixin for-desktop {
+  @media screen and (min-width: $desktop-min) {
+    @content;
+  }
+}
+#thank-you {
+  box-sizing: border-box;
+  @include for-desktop {
+    max-width: 1240px;
+    margin: auto;
+  }
+}
+
+.thank-you {
```

Comment:
@aniamusial you don't need this prefix :) just use .wrapper, .banner, heading, .paragraph 👍